### ISSUE TITLE
Propagate module config through imports

### ISSUE THREAD

**Issue by kuwv** [PyCQA/pylint#8398]
### Current problem

Many projects may require specific omissions from pylint that would require each consuming project to re-implement it's recommend settings. This is very difficult to do consistently.

### Desired solution

Mypy uses a `py.typed` placeholder to signal to itself that the project is supported. It would be useful if pylint allowed such a file to use to propagate pylint settings for developer intended use of the module.

**Comment by Pierre-Sassoulas** [comment]
I think the full configuration does not need to be propagated (a lib disabling missing-module-docstring does not mean you have to disable it too). Moreover merging everything from multiple libs would be impossible. On select options, that are easily merged with list addition like 'generated-members' though it would be a solution to a problem we currently solve with pylint plugins.

**Comment by kuwv** [comment]
@Pierre-Sassoulas Yeah, I agree. I was conceiving this an alternative to doing a separate plugin that would need to be added.
